lib/check-tags.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const log = require('./out')({});
+const _ = require('lodash');
+
+function checkTags(app, mod, name) {
+  // Returns true if the module should be skipped
+
+  if ((app.excludeTags && !mod.tags) || (!app.includeTags && !app.excludeTags)) {
+    return false; // No checks to run.
+  } else if (app.includeTags && !mod.tags) {
+    return true; // No tags for this module
+  }
+
+  if (typeof mod.tags === 'string') {
+    mod.tags = mod.tags.split(','); // If mod.tags is a single field
+  }
+
+  let excludeTagMatches = _.intersection(app.excludeTags, mod.tags);
+
+  if (excludeTagMatches.length) {
+    log.info(name,
+        `skipped as these excludeTags matched: ${excludeTagMatches}`);
+    return true; // We matched an excludeTag.
+  } else if (app.excludeTags) {
+    log.info(`${name} will run as no excludeTags matched`);
+    return false; // We did not match an excludeTag.
```

Comment:
I guess it makes sense to allow `excludeTags` and `includeTags`.

If you have:

```json
"both": {
  "tags": ["a", "b"]
},
"aOnly": {
  "tags": ["a"]
},
"bOnly": {
  "tags": ["b"]
},
"neither": {
  "tags": []
}
```

| arguments | Modules run |
|--- | --- |
| `` | `both badOnly goodOnly neither` |
| `--includeTags=a` \| `--excludeTags=b` | `both aOnly` |
| `--excludeTags=a` \| `--includeTags=b` | `neither bOnly` |
| `--includeTags=a --excludeTags=b` | `bOnly` |